steam_deploy.sh

```diff
@@ -56,13 +56,23 @@ echo ""
 
 mkdir BuildOutput
 
+steamdir=$STEAM_HOME
+manifest_path=$(pwd)/manifest.vdf
+contentroot=$(pwd)/$rootPath
+if [[ "$OSTYPE" == "darwin"* ]]; then
+  steamdir="$HOME/Library/Application Support/Steam"
+elif [[ "$OSTYPE" == "msys"* ]]; then
+  manifest_path=$(cygpath -w "$manifest_path")
+  contentroot=$(cygpath -w "$contentroot")
+fi
```

Comment:
Thinking more on it, maybe this is the only change required:

```suggestion
elif [ "$OSTYPE" == "linux" ]; then
  steamdir="/home/runner/Steam"
fi
```